packages/uicore/src/components/ListHeader/ListHeader.css

```diff
@@ -8,3 +8,7 @@
 .listHeader {
   margin: var(--spacing-medium) var(--spacing-xxlarge) !important;
 }
+
+.listHeaderStory {
+  width: 100% !important;
+}
```

Comment:
This css is only used for story. It would be better to use inline style or to create List Header.stories.css

Reply:
Done